	# This patch will actually be somewhat short lived as it's really
	# somewhat of a hack. The toolchain folks (alanm) have a set of patches
	# to remove the use of the '.' form in ppc64 assembler
--- configure
+++ configure
@@ -19253,8 +19253,8 @@
 for tmp_underscore in "" "_"; do
   cat >conftes2.s <<EOF
       	$gmp_cv_asm_text
-	$gmp_cv_asm_globl ${tmp_underscore}underscore_test
-${tmp_underscore}underscore_test$gmp_cv_asm_label_suffix
+	$gmp_cv_asm_globl .${tmp_underscore}underscore_test
+.${tmp_underscore}underscore_test$gmp_cv_asm_label_suffix
 EOF
   case $host in
   *-*-aix*)
